### PR TITLE
Documenting USS drag and drop feature

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Documenting the new ZE drag and drop feature available in Zowe V3

List the file(s) included in this PR: ze-working-with-uss-files.md +

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/ze-working-with-uss-files.md [hunk 1]
```diff
@@ -16,6 +16,21 @@
...
+
+To select one or more files or folders in the tree and drag them to a folder or session to relocate them:
+
+1. Expand **UNIX SYSTEM SERVICES (USS)** in the **Side Bar**.
+2. Select the files or folders you want to move.
+   - Select multiple items by pressing the `Ctrl` key as you select each item.
+   - Select a continuous range of items by selecting the first item in the range. Then hold down the `Shift` key until selecting the last item in the range.
+3. Click and hold the region containing the selected items to start the drag process.
+4. As the items are selected, move your cursor to the desired folder or session. Release the mouse button to start the move operation.
+   - If the move operation might overwrite one or more files, an “overwrite confirmation" message displays. Select **Confirm** to relocate these items. Select **WHAT** to stop the move operation.
```
Comment (anaxceron) [inline docs/user-guide/ze-working-with-uss-files.md:29]: @traeok Does the UI say **Confirm** or **Continue**? And what option does the UI provide for the user to cancel the move operation?

Comment (traeok) [reply docs/user-guide/ze-working-with-uss-files.md:29]: The options for the UI are "Confirm" and "Cancel". The message says: "X is opened and has pending changes in the editor. By selecting 'Confirm', any unsaved changes will be lost." 

(X being the name of the file)

Comment (anaxceron) [reply docs/user-guide/ze-working-with-uss-files.md]: @traeok -- updated the doc to reflect this -- thank you!

**Review by traeok — APPROVED** [review]
Looks great - thanks Ana!

**Pull request merged**